test_library.py

```diff
@@ -0,0 +1,111 @@
+import unittest
+
+from Library import *
+
+class TestStringMethods(unittest.TestCase):
+  def setUp(self):
+    self.library = Library([],[],[])
+
+  def test_new_library(self):
+    self.assertEqual(len(self.library.series), 0)
+    self.assertEqual(len(self.library.series), 0)
```

Comment:
i think the same code is tested twice here

Reply:
maybe it should be library.movies?

Reply:
Good catch, that was a typo! It's now addressed

Reply:
Good catch, that was a typo! It's now addressed